### PR TITLE
feat: add card component

### PR DESCRIPTION
**Pull request by irfan-nawaz**
added cards and did minor changes in spacing and positioning
live pre-view link: [https://1lgvq.csb.app/](url)

### CODE REVIEW

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
all the card vairants have box shadow , prd requirement is that only one variant should have box shadow , mention name of the variants for each card. the dismiss buttons should be on the top right corner

**Comment by irfan-nawaz** [comment]
> all the card vairants have box shadow , prd requirement is that only one variant should have box shadow , mention name of the variants for each card. the dismiss buttons should be on the top right corner

hey, Mohammed, I have read the requirements on LMS It is written, cards i.e multiple cards can have a box-shadow, and yes I have tried the cards without keeping and box-shadow but I still prefer that all the cards should have box-shadow, and also the same with dismiss icon I have tried different positions and I intended to keep them there only, as there no specific position mentioned for them on LMS and it is free to the creator,
thanks for your valuable suggestions. I kindly request you to approve if the above ones are only the cases.
thank you

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+.content {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.content-row {
+    display: flex;
+    flex-direction: row;
```
Comment (kart-c) [inline Components/Card/card.css]: Here you can remove `flex-direction: row` as its the default value

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    opacity: 1;
+    z-index: 1;
+    filter: blur(0px);
+}
+
+/* in-stock colors */
+
+#colors {
+    display: flex;
+    flex-direction: row;
```
Comment (kart-c) [inline Components/Card/card.css]: I would suggest the same here

File: Components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    justify-content: flex-start;
```
Comment (kart-c) [inline Components/Card/card.css]: This is also a default value. Attaching link for reference 

https://css-tricks.com/almanac/properties/j/justify-content/#:~:text=flex%2Dstart%20(default)%3A%20items%20are%20packed%20toward%20the%20start%20line

File: index.html [hunk 4]
```diff
@@ -107,6 +108,182 @@
...
+        </figure>
+        <div class="about">
+            <h5 class="product-description">Sustainable Materials</h5>
+            <p></p>
+            <p>Unisex</p>
+            <p>available in colors</p>
+            <div id="colors">
+                <div id="blue"></div>
+                <div id="green"></div>
+                <div id="red"></div>
```
Comment (kart-c) [inline index.html]: `id` attribute is supposed to be unique. If you want to use it multiple times you should use `class`

**Review by kart-c — CHANGES_REQUESTED** [review]

File: Components/Card/card.css (hunk 1 above)
Comment (irfan-nawaz) [reply Components/Card/card.css]: 👍🏼

File: Components/Card/card.css (hunk 2 above)
Comment (irfan-nawaz) [reply Components/Card/card.css]: 👍🏼

File: Components/Card/card.css (hunk 3 above)
Comment (irfan-nawaz) [reply Components/Card/card.css]: 👍🏼

File: index.html (hunk 4 above)
Comment (irfan-nawaz) [reply index.html]: hey Kartikeya thanks for your reviews will make the changes right away 🚀

**Review by kart-c — APPROVED** [review]

**Review by AnkurChunekar — APPROVED** [review]

**Review by starver20 — APPROVED** [review]

**Pull request merged**